architecture/topics/nsxt.adoc

```diff
@@ -0,0 +1,11 @@
+The VMware NSX-T Data Center provides a policy-based overlay network reproducing the complete set of Layer 2
+through Layer 7 networking services (such as switching, routing, access control, fire-walling, and QoS) in software for
+native {product-title} networking capabilities.
+
+The NSX-T components can be installed and configured as part of the Ansible installation procedure which integrates an {product-title} SDN
+into a datacenter-wide NSX-T virtualised network connecting bare metal, virtual machines, and {product-title} pods.
+See the xref:../../install_config/configuring_nsxtsdn.adoc#install-config-configuring-nsxt-sdn[Installation] section for information on how to install and deploy OpenShift Container Platform with VMware NSX-T.
+
+The *NSX-T Container Plug-In* (NCP) integrates {product-title} into an NSX-T Data Center, which is typically configured for the entire datacenter. 
```

Comment:
s/datacenter/data center

Reply:
"NSX-T Data Center" is the product name. :)